src/components/TopBar/AboutDialog.js

```diff
@@ -186,72 +285,47 @@ const AboutDialog = ({
                             </Alert>
                         </Collapse>
                     )}
-                <Box sx={{ height: '5em' }}>
-                    <Box
-                        sx={{
-                            display: 'flex',
-                            alignItems: 'center',
-                            justifyContent: 'center',
-                        }}
-                    >
+                <Box sx={styles.mainSection}>
+                    <Box sx={styles.logoSection}>
                         <LogoText
                             appName="Suite"
                             appColor={theme.palette.grey['500']}
                         />
                     </Box>
-                    <Grid
-                        container
-                        component="dl"
-                        columnSpacing={1}
-                        justifyContent="center"
-                        alignItems="center"
-                    >
-                        {loadingGlobalVersion && (
-                            <CircularProgress size="1.2rem" />
-                        )}
-                        {!loadingGlobalVersion && (
-                            <Grid
-                                item
-                                sx={{
-                                    display: 'flex',
-                                    alignItems: 'baseline',
-                                }}
-                            >
-                                <FormattedMessage
-                                    id="about-dialog/version"
-                                    sx={{
-                                        display: 'inline-block',
-                                        lineHeight: '1',
-                                        fontSize: '1em',
-                                    }}
-                                />
+                    <Box component="dl" sx={styles.mainInfos}>
+                        <Fade
+                            in={loadingGlobalVersion}
+                            appear
+                            unmountOnExit
+                            onExited={(node) => {
+                                if (open && !loadingGlobalVersion) {
```

Comment:
```suggestion
                                if (open) {
```
`loadingGlobalVersion` is already false no ?

Reply:
no, you can close the dialog while it still is loading

Reply:
Ok but then what's the point of intering here because we want to execute this code only when `open` is true ?